sqls/test5.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+    column_1,
+    column_2,
+        column_3,
```

Comment:
🚫 **[sqlfluff]** <[LT02](https://docs.sqlfluff.com/en/stable/rules.html#rule-LT02)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[layout.indent] - Expected indent of 4 spaces.
```suggestion
replace /     column_3,
```